multiqc/modules/fastqc/fastqc.py

```diff
@@ -268,14 +286,23 @@ def fastqc_general_stats(self):
             "format": "{:,.0f}",
         }
         headers["avg_sequence_length"] = {
-            "title": "Read Length",
+            "title": "Average Read Length",
             "description": "Average Read Length (bp)",
             "min": 0,
             "suffix": " bp",
             "scale": "RdYlGn",
             "format": "{:,.0f}",
             "hidden": hide_seq_length,
         }
+        headers["median_sequence_length"] = {
+            "title": "Median Read Length",
+            "description": "Median Read Length (bp)",
+            "min": 0,
+            "suffix": " bp",
+            "scale": "RdYlGn",
+            "format": "{:,.0f}",
+            "hidden": True,
```

Comment:
I'm happy for the median to be the one that is shown by default (I'd prefer this actually).

I was mostly thinking about the `multiqc_data` outputs for the backwards compatibility to be honest.

Reply:
```suggestion
            "hidden": hide_seq_length,
```